### DISCUSSION TITLE
Make Editorial Comments required

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Planner#1357] [Feature Requests]
> We would like to require Editors to leave a comment for the approver. This could include a comment on what was changed, why it was changed, or anything else.

**Comment by olatechpro** [comment]
@stevejburge I'm adding this note here in case we reach this issue again since it's moved to future milestone

> Editorial comment can only be added after post has been published. So, wondering how making it required will work? 
> Which is logical since comment can only be added to created post
> Maybe the term publish is wrong but let say post has to leave auto-draft

![comment](https://user-images.githubusercontent.com/46832636/196481996-ff896347-b4f9-4eac-a640-39b7308a68e4.jpg)

**Comment by stevejburge** [comment]
@olatechpro Agreed, too many complications for now
